x/committee/types/msg_test.go

```diff
@@ -56,12 +56,17 @@ func TestMsgVote_ValidateBasic(t *testing.T) {
 	}{
 		{
 			name:       "normal",
-			msg:        MsgVote{5, addr},
+			msg:        MsgVote{5, addr, Yes},
+			expectPass: true,
+		},
+		{
```

Comment:
Does it make sense to update validate basic to verify that the vote input is valid? If so, could add a test case like

```
{
  name:       "invalid vote",
  msg:        MsgVote{5, addr, 3},
  expectPass: false,
},
```

Reply:
https://github.com/Kava-Labs/kava/pull/900/commits/800604b780bb0b4a1bd452d61285ca28505a93d3